### ISSUE TITLE
🚀[FEA]: Shall I use HRRR datasets in natural/hybrid model levels when training the StormCast model?

### ISSUE THREAD

**Issue by Steven04453X** [NVIDIA/physicsnemo#873]
### Is this a new feature, an improvement, or a change to existing functionality?

Change

### How would you describe the priority of this feature request

Medium

### Please provide a clear description of problem you would like to solve.

Dear StormCast Team,
I am currently working on training the StormCast model using the HRRR dataset over the western US. Should I train the model using the HRRR data at the natural/hybrid model levels? I noticed that the input state weather variables are provided at these levels (https://catalog.ngc.nvidia.com/orgs/nvidia/teams/modulus/models/stormcast-v1-era5-hrrr#:~:text=Input%20state%20weather,refc). What if I use the data at pressure levels to train the model?

Additionally, could you please provide an estimate of the computational cost associated with training the model?

Thank you for your time to read and address my questions!

**Comment by pzharrington** [comment]
Hi @Steven04453X -- the model was originally trained on the natural/hybrid model levels, so that is technically the correct type to use if you want to remain as close as possible to the original work. It probably would work ok with pressure levels instead, but I expect the hybrid levels would give you slightly superior accuracy/behavior over higher terrain and mountains (this is conjecture, though, we have not run any experiments with pressure level data).

For an estimate of compute costs, the [paper](https://arxiv.org/abs/2408.10958) mentions:

> We train the network for a total of 450,000 training steps with batch size 64, corresponding to 29 million
noised samples seen in training. The training pipeline completes in 120 hours on 64 NVIDIA H100 GPUs

**Comment by Steven04453X** [comment]
Hi @pzharrington 
Thank you so much for your answers! I truly appreciate your help.
I will proceed with training the model on the natural/hybrid model levels. Besides, I will discuss the compute costs with my project manager.

**Comment by pzharrington** [comment]
No problem, happy to help. I will close this for now but feel free to reply again if you have other questions.
